de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/IBPartnerDAO.java

```diff
@@ -303,4 +303,6 @@ I_C_BPartner_Location retrieveBillToLocation(Properties ctx,
 	List<GeographicalCoordinatesWithBPartnerLocationId> getGeoCoordinatesByBPartnerIds(Collection<BPartnerId> bpartnerIds);
 
 	List<GeographicalCoordinatesWithBPartnerLocationId> getGeoCoordinatesByBPartnerLocationIds(Collection<Integer> bpartnerLocationRepoIds);
+
+	BPartnerLocationId retrieveCurrentBillLocationOrNull(BPartnerId partnerId);
```

Comment:
Not 100% sure if this case warants it's own BL-method, but i think it's at any rate OK for now

Reply:
I wrote this method because the old method that was doing this thing was marked as deprecated. : de.metas.bpartner.service.IBPartnerDAO.retrieveBillToLocation(Properties, int, boolean, String) . 
Maybe in the future we'll delete the old, deprecated one and use my method everywhere.